### ISSUE TITLE
Problem with deglitching the data after actualisation

### ISSUE THREAD

**Issue by wikicupcake** [xraypy/xraylarch#431]
Hello,

After the actualisation to the new version of XAS Viewer (which also forced me to uninstall and reinstall whole Larch package, since the program crushed after attempts to only actualise it), I found the problem with deglitching the data.

I wanted firstly to remove only one point from my XAS spectrum (I picked the point from the spectrum and clicked "remove point"), what resulted in removing of the nearly whole spectrum after saving the result, leaving the range between the first point and this one that I wanted to erase. I tried also with removing the chosen range (exactly this one that was left in the previous case), what was also impossible, because again it saved only the region between the first point and the point I wanted to remove (I left the parameter above). I tried to change the regions, and points, but the results are the same - removing whole spectra and leaving very little range which I wanted to remove not save.

I would be really grateful for help - what maybe I did wrong or maybe for looking if something is wrong in the actualisation file. 
I used the previous version and it was working just fine when I did the same operations.

Thanks in advance for help.

**Comment by newville** [comment]
@wikicupcake Oh, yes, very sorry for that.  This is now "fixed on github" and if you are brave you could install the latest devel version with
```
  python -m pip install https://millenia.cars.aps.anl.gov/xraylarch/downloads/xraylarch-latest-py3-none-any.whl
```
which does fix this bug with deglitching.

**Comment by avalon1774** [comment]
Hello!, 

I seem to be having a very similar but slightly different issue.  XAS viewer has recently prompted me to upgrade to larch 0.9.68. Unfortunately, something broke during the upgrade and I had to reinstall larch into a fresh conda environment with Python 3.9. Everything looked great, but my installation's deglitching functionality seems to be broken. No matter if I want to remove a point or an entire part of the energy range, the current new spectrum contains only the first two points of the dataset. 

In addition, the overwrite de-glitched data under the same group I remember from using the software before, appears to be gone, but that might be by design, I just noticed it missing in the same window I was having trouble with. 

Unfortunately, I wasn't able to install the latest developer version due to the: I
mportError: cannot import name 'help_function' from 'larch.helper' 

I am just wondering if I did something wrong during installation or set-up of larch?
